test/integration_test/extender_test.go

```diff
@@ -146,3 +151,65 @@ func driverNodeErrorTest(t *testing.T) {
 
 	destroyAndWait(t, ctxs)
 }
+
+func pvcOwnershipTest(t *testing.T) {
+	ctxs, err := schedulerDriver.Schedule(generateInstanceID(t, "ownershiptest"),
+		scheduler.ScheduleOptions{AppKeys: []string{"mysql-repl-1"}})
+	require.NoError(t, err, "Error scheduling task")
+	require.Equal(t, 1, len(ctxs), "Only one task should have started")
+
+	err = schedulerDriver.WaitForRunning(ctxs[0], defaultWaitTimeout, defaultWaitInterval)
+	require.NoError(t, err, "Error waiting for pod to get to running state")
+
+	scheduledNodes, err := schedulerDriver.GetNodesForApp(ctxs[0])
+	require.NoError(t, err, "Error getting node for app")
+	require.Equal(t, 1, len(scheduledNodes), "App should be scheduled on one node")
+
+	volumeNames := getVolumeNames(t, ctxs[0])
+	require.Equal(t, 1, len(volumeNames), "Should have only one volume")
+
+	verifyScheduledNode(t, scheduledNodes[0], volumeNames)
+
+	for _, spec := range ctxs[0].App.SpecList {
+		if obj, ok := spec.(*storage_api.StorageClass); ok {
+			if err := k8s.Instance().DeleteStorageClass(obj.Name); err != nil {
+				require.NoError(t, err, "Error deleting storage class for mysql.")
+			}
+		}
+		if obj, ok := spec.(*v1.PersistentVolumeClaim); ok {
+			updatePVC, _ := k8s.Instance().GetPersistentVolumeClaim(obj.Name, obj.Namespace)
+			if _, hasKey := updatePVC.Annotations["storage-provisioner"]; hasKey {
+				delete(updatePVC.Annotations, "storage-provisioner")
+			}
+			if _, err = k8s.Instance().UpdatePersistentVolumeClaim(updatePVC); err != nil {
+				require.NoError(t, err, "Error updating annotations in PVC.")
+			}
+		}
+	}
+
+	err = volumeDriver.StopDriver(scheduledNodes, false)
+	require.NoError(t, err, "Error stopping driver on scheduled Node %+v", scheduledNodes[0])
+
+	time.Sleep(3 * time.Minute)
+
+	var errUnscheduledPod bool
+	var depPods []v1.Pod
+	for _, spec := range ctxs[0].App.SpecList {
+		if obj, ok := spec.(*apps_api.Deployment); ok {
+			if obj.Name == "mysql" {
+				depPods, err = k8s.Instance().GetDeploymentPods(obj)
+				require.NoError(t, err, "Error getting pods for deployment ,mysql.")
+			}
+		}
+		for _, pod := range depPods {
+			for _, cond := range pod.Status.Conditions {
+				if cond.Type == v1.PodScheduled && cond.Status == v1.ConditionFalse {
+					errUnscheduledPod = true
+				}
+			}
+		}
+	}
+	require.Equal(t, false, errUnscheduledPod, "Pod should not have been schedule.")
```

Comment:
Expected is `true`, I think you didn't push your change.

Reply:
Fixed.